### PR TITLE
EDUCATOR-5518 - remove axios

### PR DESCRIPTION
**Pull request by mattcarter**
Remove axios as a dependency from package.json and rebuild the javascript dependancies
Bump the ORA version to 2.13.8

**TL;DR -** Remove unneeded dependency on `axios` from ORA

JIRA: [JIRA-5518](https://openedx.atlassian.net/browse/JIRA-5518)

**What changed?**

- removed axios -- appears as side-effect addition to package.json when converting to webpack
- other minor version changes in dependancies 

**Developer Checklist**

- [x] Reviewed the [release process](./release_process.md)
- [ ] Translations and JS/SASS compiled
- [x] Bumped version number in [setup.py](../setup.py) and [package.json](../package.json)

**Testing Instructions**

- passes unit tests
- general functionality tests

**Reviewer Checklist**

Collectively, these should be completed by reviewers of this PR:

- [x] I've done a visual code review
- [x] I've tested the new functionality

FYI: @edx/masters-devs-gta

### CODE REVIEW

**Review by jnlapierre — APPROVED** [review]
LGTM

**Review by nsprenkle — APPROVED** [review]
With the caveat that I haven't tested with the updated dependencies, local testing solely with `axios` removed didn't turn up any weird behavior on my end.
